src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.CommonSVTControls.ps1

```diff
@@ -92,6 +96,148 @@ class CommonSVTControls: ADOSVTBase {
         }
        return $controlResult
     }
+    hidden [ControlResult] CheckBuildServiceAccessOnBranch([ControlResult] $controlResult)
+    {
+        $controlResult.VerificationResult = [VerificationResult]::Failed
+        try
+        {
+            $isControlPassing = $true
+            $projectId = ($this.ResourceContext.ResourceId -split "project/")[-1].Split('/')[0]           
+            $url = 'https://dev.azure.com/{0}/_apis/Contribution/HierarchyQuery?api-version=5.0-preview.1' -f $($this.OrganizationContext.OrganizationName);
+            $inputbody = '{"contributionIds":["ms.vss-admin-web.security-view-members-data-provider"],"dataProviderContext":{"properties":{"permissionSetId": "2e9eb7ed-3c0a-47d4-87c1-0ffdd275fd87","permissionSetToken":"","sourcePage":{"url":"","routeId":"ms.vss-admin-web.project-admin-hub-route","routeValues":{"project":"","adminPivot":"repositories","controller":"ContributedPage","action":"Execute"}}}}}' | ConvertFrom-Json
```

Comment:
keep this "2e9eb7ed-3c0a-47d4-87c1-0ffdd275fd87" at class variable and use that variable here